spec/models/mixins/aggregation_mixin_spec.rb

```diff
@@ -41,14 +37,10 @@
 
     it "calculates from objects" do
       cluster = cluster_2_1_host(hardware_args)
-      partial_cluster = cluster.hosts[1..1] + cluster.hosts[2..2]
-      expect(cluster.aggregate_cpu_total_cores(partial_cluster)).to eq(8)
-    end
+      partial_cluster = Host.where(:id => cluster.hosts[1..2].map(&:id))
```

Comment:
So, this isn't your fault, but I almost want to delete this spec (the file, not this particular example), and create a rspec shared context that all the models that `include AggregationMixin` have in their specs to confirm it is working.  Because right now, this just tests this for `Host`, which is kinda pointless for a module since it is one of many models that include this module.

It seems like you should be doing something like what you did in `ext_management_system_spec.rb` above, but using `described_class` or something similar to make it generic to confirm that this has the desired effect across all classes that include this module.